conf/profile.conf.js

```diff
@@ -49,6 +49,14 @@ exports.config = {
           'IEDriverServer_Win32_${connectionConfigs.direct.binaries.iedriver.version}.' + '${connectionConfigs.direct.binaries.iedriver.patch}.zip',
           executable: 'IEDriverServer.exe'
         },
+        // edge based on chromium; Note: If on a Windows 64bit system the variable osTypeString returns win32 string (see uiveri5.js), then maintain the url below with e.g. edgedriver_win64.zip 
+        edgedriver: {
+          version: '80.0.361.50',
```

Comment:
why not use the auto-update mechanism, similar to chrome ?

Reply:
Microsoft does not have the latest Edge version (80, 81, 82) under one location as in chromedriver.storage; 
See https://msedgewebdriverstorage.z22.web.core.windows.net the latest Edge versions (80, 81, 82) are not here.

Reply:
Also, I have tested and if I delete the msedgedriver.exe from npm\node_modules\@ui5\uiveri5\selenium\ when a new uiveri5 execution is started, the driver is downloaded from Microsoft web site, but the version is the one from profile.conf.js (e.g. 80.0.361.50); 
Let me know if you have any suggestion on how to detect the latest version from Microsoft, for now looks like there is a limitation.

Reply:
add latestVersionFileUrl pointing to the driverVersions.json

Reply:
ok, thanks

Reply:
so, I will add the complete version (major+minor) in driverVersions.json since there is no mechanism to get the latest minor version from major version on Microsoft side.

Reply:
please verify latest changes about adding latestCompleteVersionFileUrl pointing to the driverVersions.json. 
Note: I had to use a different name 'latestCompleteVersionFileUrl' , to differentiate in if statements between Major and Complete version from file.